cylc/rose/utilities.py

```diff
@@ -87,7 +87,23 @@ def get_rose_vars_from_config_node(config, config_node, environ):
     for section in ['env', templating]:
         # Add standard ROSE_VARIABLES
         config_node[section].set(['ROSE_SITE'], rose_site)
+        if 'ROSE_VERSION' in config_node[section].value:
+            user_rose_version = config_node[section].value['ROSE_VERSION']
+            LOG.warning(
+                f'[{section}]ROSE_VERSION={user_rose_version.value} '
+                'from rose-suite.conf will be ignored: '
+                f'ROSE_VERSION set: {ROSE_VERSION}'
+            )
         config_node[section].set(['ROSE_VERSION'], ROSE_VERSION)
+
+        if 'CYLC_VERSION' in config_node[section].value:
+            user_cylc_version = config_node[section].value['CYLC_VERSION']
+            LOG.warning(
+                f'[{section}]CYLC_VERSION={user_cylc_version.value} '
+                'from rose-suite.conf will be ignored.'
```

Comment:
If we are reporting the Rose version we might as well report the Cylc version too:

```suggestion
                'from rose-suite.conf will be ignored: '
                f'Using Cylc: {CYLC_VERSION}'
```

Just need to add an import at the top:

```diff
 from cylc.flow.hostuserutil import get_host
-from cylc.flow import LOG
+from cylc.flow import (
+    LOG,
+    __version__ as CYLC_VERSION,
+)
 from cylc.rose.jinja2_parser import Parser
```

Reply:
Oh, just spotted:

> Certainly not here, since we only unset it.

From the perspective of Cylc Rose that makes sense.

From the perspective of the user, they asked for "a" but got "b" so it might be a bit confusing not to report "b" in this case?

Although they will find out soon enough when they run their workflow...